gradle.properties

```diff
@@ -17,7 +17,7 @@ artifactoryPublishingMavenRepo=https://zowe.jfrog.io/zowe/libs-release-local
 artifactoryPublishingMavenSnapshotRepo=https://zowe.jfrog.io/zowe/libs-snapshot-local
 
 # Artifacts version
-version=1.28.14
+version=1.28.14-SNAPSHOT
```

Comment:
Is this change intentional?

Reply:
yes, otherwise publish scripts will not update the version properly